uiowa/uiowa
# Event Locations Probe - Autofill edition

Relates to https://github.com/uiowa/uiowa/issues/6494

# To do if it moves forward:

- Look into adding chosen or select2 to make dropdown selection easier
- Create a array map of api to field to autofill more efficiently
- Public view display of places on content hub with a webform to add/submit change requests
- Decide on whether dropdown should reset or store the location.
- Should the dropdown go disabled until the location fields are reset (js) to prevent overwriting?

# To Test

```
ddev composer install && ddev blt ds --site=oniowa.uiowa.edu && ddev drush @oniowa.local uli /node/add/event
```

See the new autofill and geolocation field. Select a location from the list. See things populate and the select reset. Feel free to edit the result. Save. Replicate is available too. That is neat.

![out](https://github.com/uiowa/uiowa/assets/4663676/4d669759-c021-4aec-a86d-89f0c764960c)

Video demo here: https://uicapture.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=6a0bf598-616b-41e9-9c5b-b01a00e3b961
